net-analyzer/nuttcp/nuttcp-8.1.4.ebuild

```diff
@@ -0,0 +1,61 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit readme.gentoo-r1 systemd toolchain-funcs
+
+DESCRIPTION="Network performance measurement tool"
+HOMEPAGE="http://www.nuttcp.net/"
+SRC_URI="http://nuttcp.net/${PN}/${P}.tar.bz2"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="ipv6 xinetd"
+
+RDEPEND="xinetd? ( sys-apps/xinetd )"
+
+DOCS=( examples.txt README )
+# Honor CC, LDFLAGS, CFLAGS, CPPFLAGS
+PATCHES=( "${FILESDIR}"/${P}-makefile.patch )
+
+DOC_CONTENTS="You may need to add these lines to /etc/services:\n
```

Comment:
*May*? How should user determine if he *needs* to actually do that?

Reply:
This is not a Tetris neither any other game. This is a software to measure network quality. Working with it requires some skills. A user who will use it should know what to do with it.
A user might like to mention nuttpc in iptables rules, for example, or in xinetd service. And this will require from him to add these lines to /etc/services.